src/cljam/algo/sorter.clj

```diff
@@ -174,25 +165,21 @@
 (defn sort-by-pos
   "Sorts alignments by chromosomal position."
   [rdr wtr & [option]]
-  (sort! rdr wtr (into (or option {}) {:mode :coordinate})))
+  (sort! rdr wtr (into (or option {}) {:mode header/order-coordinate})))
 
 (defn sort-by-qname
   "Sort alignments by query name."
   [rdr wtr & [option]]
-  (sort! rdr wtr (into (or option {}) {:mode :queryname})))
+  (sort! rdr wtr (into (or option {}) {:mode header/order-queryname})))
 
 (defn sorted-by?
   "Returns true if the sam is sorted, false if not. It is detected by
   `@HD SO:***` tag in the header."
   [rdr]
-  (let [so (:SO (:HD (sam/read-header rdr)))]
-    (or (= so (name order-queryname))
-        (= so (name order-coordinate)))))
+  (header/sorted-by? (sam/read-header rdr)))
```

Comment:
Though it is not related to this PR, why was this function named `sorted-by?` ...? `-by` seems unnecessary.

Reply:
I agree. Renamed it to `sorted?`.